services/venice-router/src/main/java/com/linkedin/venice/router/stats/AggRouterHttpRequestStats.java

```diff
@@ -200,6 +200,11 @@ public void recordCompressedResponseSize(String storeName, double compressedResp
     recordStoreStats(storeName, stats -> stats.recordCompressedResponseSize(compressedResponseSize));
   }
 
+  public void recordDecompressedResponseSize(String storeName, double compressedResponseSize) {
+    totalStats.recordDecompressedResponseSize(compressedResponseSize);
+    recordStoreStats(storeName, stats -> stats.recordDecompressedResponseSize(compressedResponseSize));
+  }
```

Comment:
```suggestion
  public void recordDecompressedResponseSize(String storeName, double decompressedResponseSize) {
    totalStats.recordDecompressedResponseSize(decompressedResponseSize);
    recordStoreStats(storeName, stats -> stats.recordDecompressedResponseSize(decompressedResponseSize));
  }
```
How about this naming?